.settings/org.eclipse.jdt.core.prefs

```diff
@@ -9,15 +9,109 @@ org.eclipse.jdt.core.codeComplete.staticFieldPrefixes=
 org.eclipse.jdt.core.codeComplete.staticFieldSuffixes=
 org.eclipse.jdt.core.codeComplete.staticFinalFieldPrefixes=
 org.eclipse.jdt.core.codeComplete.staticFinalFieldSuffixes=
+org.eclipse.jdt.core.compiler.annotation.inheritNullAnnotations=disabled
+org.eclipse.jdt.core.compiler.annotation.missingNonNullByDefaultAnnotation=ignore
+org.eclipse.jdt.core.compiler.annotation.nonnull=org.eclipse.jdt.annotation.NonNull
```

Comment:
For more information on these annotations, see our [FindBugs page](http://jmri.sourceforge.net/help/en/html/doc/Technical/FindBugs.shtml)

The first two of the ones in the comment above are `javax.annotation.CheckForNull` and `javax.annotation.Nonnull` respectively. 

 For `@NonNullByDefault`, I think the right choice is `javax.annotation.ParametersAreNonnullByDefault` but I haven't tested that myself.